mkdocs_static_i18n/plugin.py

```diff
@@ -78,7 +81,31 @@ def on_nav(self, nav, config, files):
         """
         Translate i18n aware navigation to honor the 'nav_translations' option.
         """
-        i18n_nav = self.reconfigure_navigation(nav, config, files)
+
+        # maybe move to another file and don't pass it as parameter?
+        class NavHelper:
+            translated_items: int = 0
+            homepage: Optional[Page] = None
+
+        i18n_nav = self.reconfigure_navigation(nav, config, files, NavHelper)
+
+        # report translated entries
+        if NavHelper.translated_items:
+            log.info(
+                f"Translated {NavHelper.translated_items} navigation element"
+                f"{'s' if NavHelper.translated_items > 1 else ''} to '{self.current_language}'"
+            )
+
+        if hasattr(i18n_nav, "homepage"):
```

Comment:
since i18n_nav is a `Navigation`, shouldn't it have a `homepage` always? This if looks useless after your refactoring

Reply:
True